registry/recipes/nginx.md

```diff
@@ -181,8 +181,6 @@ Review the [requirements](/registry/recipes/index.md#requirements), then follow
 
     registry:
       image: registry:2
-      ports:
-        - 127.0.0.1:5000:5000
```

Comment:
Without any ports published, the registry won't me accessible, which makes it ... not very useful?

Reply:
One would access the registry via the nginx container.

authorized User -> nginx -> registry
unauthorized User -> nginx -> return 401

Reply:
Ah, you're right, sorry, read too fast